activiti-core/activiti-api-impl/activiti-api-process-runtime-impl/src/test/java/org/activiti/runtime/api/impl/ExtensionsVariablesMappingProviderTest.java

```diff
@@ -1111,4 +1100,23 @@ public void addCustomFunctions(ActivitiElContext elContext) {
             }
         }
     }
+
+    @Test
+    void should_setMappingEphemeral_basedOn_mappingIsEphemeralOrNot() throws IOException {
+        //given
+        ProcessExtensionModel extensions = OBJECT_MAPPER.readValue(
+            new File("src/test/resources/task-variable-mapping-extensions-with-ephemeral.json"),
+            ProcessExtensionModel.class
+        );
+        Extension processExtensions = extensions.getExtensions("Process_taskVarMapping");
+        DelegateExecution executionEphemeralTask = buildExecution(processExtensions, "ephemeralTask");
+        DelegateExecution executionNonEphemeralTask= buildExecution(processExtensions, "nonEphemeralTaskOne");
+        DelegateExecution executionAnotherNonEphemeralTask= buildExecution(processExtensions, "nonEphemeralTaskTwo");
```

Comment:
May I suggest to rename to `implicitNonEphemeralTask` and `explicitNonEphemeralTask`?